Composer/packages/adaptive-form/src/components/ExpressionSwitchWindow.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
+import React from 'react';
+import formatMessage from 'format-message';
+
+const styles = {
+  fieldTypeText: css`
+    height: 32px;
+    font-style: italic;
+    color: #8a8886;
+    padding: 0 4px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    font-size: 15px;
+  `,
+  switchToExpressionText: css`
+    height: 32px;
+    cursor: pointer;
+    padding: 0 4px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    font-size: 15px;
+  `,
+  icon: css`
+    margin-right: 5px;
+  `,
+};
+
+export const ExpressionSwitchWindow = (props: { type: string; onSwitchToExpression: () => void }) => {
+  const { type, onSwitchToExpression } = props;
+  return (
+    <React.Fragment>
+      <div css={styles.fieldTypeText}>{`Start typing ${type} or`}</div>
+
+      <div
+        css={styles.switchToExpressionText}
+        onClick={() => {
+          onSwitchToExpression();
+        }}
+      >
+        <FontIcon css={styles.icon} iconName={'CalculatorEqualTo'} />
+        {formatMessage(`Write in expression`)}
```

Comment:
Write expression or Write an expression - Write in expression sounds awkward.  Usually you fill in something, not write in - except when voting.

Reply:
This is what's in the designs: 
https://www.figma.com/file/ISGuziIavuIZdwOHTx3G1u/Property-field?node-id=1%3A1726
![image](https://user-images.githubusercontent.com/66701106/96322405-c4d5f280-0fcd-11eb-99c7-655b2794a6a4.png)
@sangwoohaan